.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -100,9 +101,19 @@ As a project onboarding into FINOS, you will need to familiarize yourself and yo
  - [FINOS Project/Standards Governance](https://community.finos.org/docs/governance)
  - [FINOS Project Lifecycle](https://community.finos.org/docs/governance/Software-Projects/project-lifecycle)
 
-## Record The Contribution (FINOS Infra)
-- [ ] (optional) Identify and Assign FINOS Strategic Advisor
-- [ ] Submit contribution to LF by opening a ticket via https://jira.linuxfoundation.org/browse/SS and marking contribution as "Exploratory"; attach a summary of the **Business Problem** and **Proposed Solution** (above) of the project.
+# Socialisation 
+- [ ] Write and send contribution proposal announcement (optional - see below)
+- [ ] Lead maintainer to send out announcement to community@finos.org using this template:
+    
+    ```
+    Dear FINOS Community, 
+    
+    We would like to propose a new FINOS project. Please review the proposal details at (_TODO: add link to the GitHub issue proposal_).
+    
+    If you're interested in participating, please :+1: the GitHub issue proposal and drop a comment with your name, org and email
+
+   Thanks a lot,
+   ```     
```

Comment:
The socialization step has been moved to take place before the TOC approval. This is something the TOC looks for prior to approval so it makes sense for it to take place before.